apps/editor/src/app/atoms/modelTitle.tsx

```diff
@@ -86,11 +100,18 @@ export function ModelTitle({
       {/* title */}
       <FlexboxGrid.Item colspan={12}>
         <FlexboxGrid align="middle">
-          <Link to={closePath}>
-            <ChevronLeft />
-          </Link>
-
-          <h1>{titleView()}</h1>
+          <FlexRow>
+            <PaddedCol xs={3}>
+              <h1>
```

Comment:
We shouldn't use h1 for this as it's not a title, we can have both {titleView} and the chevron in it though

Reply:
H1 would be semantically correct though, so we should change the size then.